translate/convert/test_po2prop.py

```diff
@@ -62,6 +62,15 @@ def test_space_preservation(self):
         print(propfile)
         assert propfile == propexpected
 
+    def test_no_value(self):
+        """check that we can handle keys without value"""
+        posource = '''#: KEY\nmsgctxt "KEY"\nmsgid ""\nmsgstr ""\n'''
+        proptemplate = '''KEY\n'''
+        propexpected = '''KEY = \n'''
```

Comment:
`propexpected` is definitely different of `proptemplate`. This is the kind of inconsistency I wanted to add tests for, and it exposes that a change `po2prop` is needed to handle this.